Gemfile

```diff
@@ -2,3 +2,9 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in action_subscriber.gemspec
 gemspec
+
+if RUBY_ENGINE == "ruby" && RUBY_VERSION.split(".").first.to_i < 2
```

Comment:
Can we bring this back into the gemspec, or do you want to leave this in the Gemfile?